common-rest/src/main/java/org/uniprot/api/rest/service/BasicSearchService.java

```diff
@@ -161,19 +160,15 @@ protected SolrRequest createSolrRequest(SearchRequest request, boolean includeFa
             SearchRequest request,
             FacetConfig facetConfig,
             AbstractSolrSortClause solrSortClause,
-            DefaultSearchHandler defaultSearchHandler,
-            boolean includeFacets) {
+            boolean includeFacets,
+            QueryBoosts queryBoosts) {
 
         SolrRequest.SolrRequestBuilder requestBuilder = SolrRequest.builder();
 
         String requestedQuery = request.getQuery();
 
         boolean hasScore = false;
```

Comment:
I think we do not use hasScore in this method, right? can we remove this line?

Reply:
it's used a few lines below:

```java
        if (solrSortClause != null) {
            requestBuilder.addSort(solrSortClause.getSort(request.getSort(), hasScore));
        }
```